tools/docker/builder/openwrt/Dockerfile

```
@@ -81,7 +81,10 @@ RUN echo "src-git prpl ${PRPL_FEED}" >> feeds.conf.default \
     && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}=y" >> .config \
     && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y" >> .config \
     && echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y" >> .config \
-    && make defconfig \
+    && if [ -n "$INTEL_FEED" ] ; then \
```

I think it would make a whole lot more sense to use
```suggestion
    && if [ "$TARGET_PROFILE" = "DEVICE_NETGEAR_RAX40" ] ; then \
```

---

Indeed now that we have it it makes a lot more sense to use it.
Thanks :+1: